app/src/main/java/org/simple/clinic/bloodsugar/unitselection/BloodSugarUnitSelectionModel.kt

```diff
@@ -2,6 +2,18 @@ package org.simple.clinic.bloodsugar.unitselection
 
 import android.os.Parcelable
 import kotlinx.android.parcel.Parcelize
+import org.simple.clinic.bloodsugar.BloodSugarUnitPreference
 
 @Parcelize
-class BloodSugarUnitSelectionModel : Parcelable
+data class BloodSugarUnitSelectionModel(
+    val bloodSugarUnitPreference: BloodSugarUnitPreference
```

Comment:
If this is added to the model, it should be updated when the selection is changed via the radio group. I don't see this being done, can we update this please?